Automattic/jetpack
# VideoPress: Fix Input component behavior

Closes #26133
Closes #26138

#### Changes proposed in this Pull Request:

* Fixes style issues
* Adds loading state
* Adds icon to clear the input

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:







* Go to the storybook: `Run cd projects/js-packages/storybook && pnpm run storybook:dev`
* On `Packages/VideoPress/Input` component, check that a `x` icon shows when the input is not of type `textarea` and there is text on it
* Check the `loading` prop
* Check the `size` prop
* With Jetpack VideoPress installed, check the dashboard use

![Screenshot_2022-09-09_18-00-52](https://user-images.githubusercontent.com/8486249/189445902-881e1f0c-84a0-487f-956f-1074c6fcb3d7.png)
![Screenshot_2022-09-09_18-01-18](https://user-images.githubusercontent.com/8486249/189445898-4d57bb07-ad06-4688-98b5-8cce83fd21f3.png)
![Screenshot_2022-09-09_18-01-40](https://user-images.githubusercontent.com/8486249/189445893-62e35c1a-391f-4916-9f81-50314fd8ce93.png)
